t/04-clone.t

```diff
@@ -45,7 +45,7 @@ $ENV{GITGOT_FAKE_GIT_WRAPPER} = 1;
   my $entry = LoadFile( $config );
   is( $entry->[0]{name} , 'fake-git-repo'              , 'expected name' );
   is( $entry->[0]{type} , 'git'                        , 'expected type' );
-  is( $entry->[0]{path} , path( "$dir/fake-git-repo" ) , 'expected path' );
+  is( $entry->[0]{path} , ''.path( "$dir/fake-git-repo" ) , 'expected path' );
```

Comment:
Why is the "append to an empty string" bit needed?

Reply:
path is a Path::tiny object, and `is` won't auto-overload it into its string representation iirc.

Reply:
ugh.

Reply:
There is `->stringify` if you want a more couth version

Reply:
BTDT - https://github.com/genehack/app-gitgot/tree/yanick/test2

Reply:
yay.